src/Controller/BookingsController.php

```diff
@@ -500,30 +500,75 @@ protected function _getUpdateParamFieldMapping()
         return [
             'start' => [
                 'field'     => 'start',
-                'transform' => [$this, '_parseIso8601'],
+                'transform' => function ($value) {
+                    if (empty($value)) {
```

Comment:
Looks like there's validation logic here. I don't think it belongs here at all. Could this be approached  differently? Perhaps by using a validator?

Reply:
Perhaps.

Reply:
Created https://github.com/RebelCode/rcmod-eddbk-rest-api/issues/7